overlore/llm/constants.py

```diff
@@ -28,12 +28,16 @@ class ChatCompletionModel(Enum):
 You must include the plot twist into the conversation to add depth.
 ${gr.complete_json_suffix_v2}"""
 
-TOWNHALL_USER = """My realm is {realm_name}\n{relevant_event}{plotline}{thoughts}Here are my NPCs: {npcs}"""
+TOWNHALL_USER = (
+    """My realm is {realm_name}\n{relevant_event}{townhall_input}{plotline}{thoughts}Here are my NPCs: {npcs}"""
+)
 
 RELEVANT_EVENT = "Here is the most interesting event for my realm: {event_string}\n"
 
 CURRENT_PLOTLINE = "This is the current plotline to follow: {plotline}\n"
 
+TOWNHALL_INPUT = "This is what the Realm's Lord gives as input to the villagers: {townhall_input}\n"
```

Comment:
`The Lord of the Realm has spoken to its villagers and says: {townhall_input}\n` no?

Reply:
Makes sense. It would be wise to notify the user that what he's giving as input is passed as the Lords speech to the villagers somehow in that case.